post/views.py

```
@@ -9,6 +9,8 @@
 from account.models import User
 from tag.models import Tag
 from account.request_serializers import SignInRequestSerializer
+from drf_yasg import openapi
+from .models import Post, Like, User
```

Ruff (0.11.9)

Redefinition of unused `Post` from line 6

Remove definition: `Post`

(F811)